include/FreeRTOSIPConfigDefaults.h

```diff
@@ -380,7 +380,7 @@
  * The default has always been 50, but a value of 1000
  * is recommended (see RFC6298) because hosts often delay the
  * sending of ACK packets with 200 ms. */
-    #define ipconfigTCP_SRTT_MINIMUM_VALUE_MS    50U
+    #define ipconfigTCP_SRTT_MINIMUM_VALUE_MS    50
```

Comment:
Yes correct, as the value will be compared with an **`int32_t`**

Reply:
@rleh thanks for this PR.

For short term I think this is okay, but since milliseconds cannot be negative, I think that the macro should be reverted back to the original value of `50U` and the variable should be modified to be `uint32_t` if it only ever holds milliseconds.